compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -211,6 +216,11 @@ impl<Prov> Scalar<Prov> {
         Scalar::Int(f.into())
     }
 
+    #[inline]
+    pub fn from_f128(f: Quad) -> Self {
+        Scalar::Int(f.into())
+    }
```

Comment:
There's `to_{f32,f64}` at the bottom of this file, we should probably have that for the new ones as well. (Can be a future PR though.)